src/instructlab/data/generate.py

```diff
@@ -11,7 +10,7 @@
     help="Name of the model used during generation.",
 )
 @click.option(
-    "--num-cpus",
+    "--num-processes",
```

Comment:
This is a functional change that's hidden in a cleanup commit. It should really be surfaced on its own. It's also breaking backwards compatibility where your design otherwise goes out of its way to ensure backwards compatibility.

Reply:
True, I will leave as cpu for now